app/controllers/v1/events_controller.rb

```diff
@@ -1,16 +1,9 @@
 class V1::EventsController < V1::BaseController
 
   def index
-    start = DateTime.parse("2015-03-14 6pm EST").utc
-    events = []
-    (1..7).each do |n|
-      events << {
-        "id":        n,
-        "starts_at": start + n.days,
-        "ends_at":   start + n.days + 2.hours,
-      }
-    end
-    render json: { events: events }, status: 200
+    @events = Event.where(starts_at: Time.now..4.weeks.from_now)
+                    .order(:starts_at).limit(10)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.